### PR TITLE
Integrate with OneTrust

### PR DESCRIPTION
**Pull request by luis30n**
Add OneTrust Cookie compliance management integration.
Use dynamic configuration depending on environment.

### CODE REVIEW

File: packages/hex-docs/templates/index.ejs [hunk 1]
```diff
@@ -10,11 +10,16 @@
...
+    <% if(process.env.ONETRUST_ID) { %>
+      <script src="https://cdn.cookielaw.org/scripttemplates/otSDKStub.js" type="text/javascript" charset="UTF-8" data-domain-script="<%- process.env.ONETRUST_ID %>"></script>
+      <script type="text/javascript"> function OptanonWrapper() { { window.dataLayer.push({ event: 'OneTrustGroupsUpdated' }); } } </script>
+    <% } %>
     <script src="<%- root %>assets/js/prism.js" data-manual></script>
     <script src="<%- root %>assets/js/all.js"></script>
     <script src="<%- root %>hex.min.js"></script>
   </head>
   <body>
+    <button id="ot-sdk-btn" class="ot-sdk-show-settings" hidden="hidden"> Cookie Settings</button>
```
Comment (Angelmmiguel) [inline packages/hex-docs/templates/index.ejs:22]: Can we make this conditional too?

**Review by Angelmmiguel — APPROVED** [review]
LGTM! Added a final comment, please check it before merging the PR :)

File: packages/hex-docs/templates/index.ejs (hunk 1 above)
Comment (luis30n) [reply packages/hex-docs/templates/index.ejs:22]: Of course!

**Review by Angelmmiguel — APPROVED** [review]
LGTM!

**Pull request merged**